src/veritysetup/veritysetup.c

```diff
@@ -89,6 +91,31 @@ static int run(int argc, char *argv[]) {
                 if (r < 0)
                         return log_error_errno(r, "Failed to configure data device: %m");
 
+#ifdef CRYPT_ACTIVATE_IGNORE_CORRUPTION
+                if (getenv_bool("SYSTEMD_CRYPT_ACTIVATE_IGNORE_CORRUPTION") > 0)
+                        flags |= CRYPT_ACTIVATE_IGNORE_CORRUPTION;
+#endif
+
+#ifdef CRYPT_ACTIVATE_RESTART_ON_CORRUPTION
+                if (getenv_bool("SYSTEMD_CRYPT_ACTIVATE_RESTART_ON_CORRUPTION") > 0)
+                        flags |= CRYPT_ACTIVATE_RESTART_ON_CORRUPTION;
+#endif
+
+#ifdef CRYPT_ACTIVATE_IGNORE_ZERO_BLOCKS
```

Comment:
The minimum libcryptsetup supported version is 2.0.1, and the following options were supported before that, so these 3 ifdefs can be removed:

CRYPT_ACTIVATE_IGNORE_CORRUPTION
CRYPT_ACTIVATE_RESTART_ON_CORRUPTION
CRYPT_ACTIVATE_IGNORE_ZERO_BLOCKS

https://gitlab.com/cryptsetup/cryptsetup/-/commit/4dd703ea6c49881a

Reply:
Is this a mandatory change? I prefer to keep these for consistency; anyway, I can remove these and keep both `CRYPT_ACTIVATE_CHECK_AT_MOST_ONCE` and `CRYPT_ACTIVATE_PANIC_ON_CORRUPTION`.

Reply:
I'm not a maintainer, so I can't mandate anything :-) But in the past, maintainers have consistently expressed a preference for keeping \#ifdefs down to a bare minimum, and for unconditionally using interfaces included in minimum supported versions of external dependencies.

Reply:
Thanks, I will remove them.